app/data/models/payment-request.js

```diff
@@ -14,6 +14,7 @@ module.exports = (sequelize, DataTypes) => {
     deliveryBody: DataTypes.STRING,
     dueDate: DataTypes.STRING,
     frn: DataTypes.BIGINT,
+    ledger: DataTypes.STRING,
```

Comment:
Need to know which ledger a request was sent to so we can restrict expected settlements to AP only